autometrics/Cargo.toml

```diff
@@ -14,6 +14,12 @@ readme = "README.md"
 
 [features]
 # Metrics backends
+metrics-0_21 = ["dep:metrics"]
+opentelemetry-0_19 = ["opentelemetry_api/metrics"]
+prometheus-0_13 = ["dep:prometheus"]
+prometheus-client-0_21 = ["dep:prometheus-client"]
```

Comment:
from what ive seen most crates use a notation like `metrics02` instead of `metrics-0_21`. whats the reason we picked it this way?

Reply:
I thought it's more readable like this, but I could be persuaded to do it the other way

Reply:
If you agree, I'd say readability > convention 😄 We'll start a new convention.